data_generator/generate_data_test.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2021 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import datetime
+
+import generate_data
+
+
+@pytest.fixture
+def github_changes():
+    return generate_data.make_changes(2, "github", 604800)
+
+
+@pytest.fixture
+def gitlab_changes():
+    return generate_data.make_changes(2, "gitlab", 604800)
+
+
+def test_changes_github(github_changes):
+
+    # example of valid output
```

Comment:
I'd love to have this as a variable and then you can do things like valid_output.keys() == github_changes.keys() .. and also write a little loop to assert the type of each key is the same.  

eg:
```
for key in output.keys():
    type(output[key]) == type(github_changes[key])
```